client/services/PlanningStoreService.js

```diff
@@ -0,0 +1,102 @@
+import { isNil } from 'lodash'
+import { createStore } from '../utils'
+import * as actions from '../actions'
+
+PlanningStoreService.$inject = [
+    '$rootScope',
+    'api',
+    'config',
+    '$location',
+    '$timeout',
+    'vocabularies',
+    'superdesk',
+    'upload',
+    'notify',
+    'privileges',
+    'notifyConnectionService',
+    'userList',
+    'desks',
+    'metadata',
+    'session',
+    'deployConfig',
+    'gettextCatalog',
+]
+export function PlanningStoreService(
+    $rootScope,
+    api,
+    config,
+    $location,
+    $timeout,
+    vocabularies,
+    superdesk,
+    upload,
+    notify,
+    privileges,
+    notifyConnectionService,
+    userList,
+    desks,
+    metadata,
+    session,
+    deployConfig,
+    gettextCatalog
+) {
+    let self = this
+
+    self.store = null
```

Comment:
just wondering why we keep the reference here, it's using the store only in controller right?
I would prefer if it would be used in controller like `const store = planningStore.createStore()`; so we don't have to deal with its reseting later